cookbooks/galera/recipes/server.rb

```diff
@@ -184,11 +189,24 @@
 end
 
 my_ip = node['ipaddress']
-init_host = galera_config['init_node']
+
+if node[:galera_config]["hostnames"] then
```

Comment:
"hostnames" for me means that it's an array or a map, please change for something like "use_hostname" or similar